tensorrt_llm/serve/openai_server.py

```
@@ -2001,8 +2010,14 @@ async def _parse_video_generation_request(
                 data["num_inference_steps"] = int(form["num_inference_steps"])
             if "guidance_scale" in form and form["guidance_scale"]:
                 data["guidance_scale"] = float(form["guidance_scale"])
+            if "guidance_scale_2" in form and form["guidance_scale_2"]:
```

Ruff (0.15.12)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)